### DISCUSSION TITLE
Can non-global parameters only be single values when using Csnippets?

### DISCUSSION THREAD

**Discussion by watermath** [kingaa/pomp#212] [Q&A]
When specifying pomp or spatpomp components I would like to have a parameter that is a vector, say rho, of parameters rho1, rho2,...etc so that in my Csnippet I can just refer to parameters by their position in rho and not have to explicitly code using rho1, rho2 etc. Is this possible? In the bm(), measles() and lorenz() example in spatpomp the parameters all refer to single values. 
I'm referring to model parameters to be estimated and NOT fixed globals like dist_C in the bm() function or V in measles(). I appreciate any help.

**Comment by kingaa** [comment]
Consult [FAQ 3.1](https://kingaa.github.io/pomp/FAQ.html#vector-in-csnippet) and let us know here if there is any remaining confusion.
